lib/apis/v3/felixconfig.go

```diff
@@ -87,6 +87,10 @@ type FelixConfigurationSpec struct {
 	// attempts to acquire the iptables lock if it is not available. Lower values make Felix more
 	// responsive when the lock is contended, but use more CPU. [Default: 50ms]
 	IptablesLockProbeInterval *metav1.Duration `json:"iptablesLockProbeInterval,omitempty" configv1timescale:"milliseconds" confignamev1:"IptablesLockProbeIntervalMillis"`
+	// FeatureDetectOverride is used to override the feature detection. Values are specified in
+	// a comma separated list. A value of "true" or "false" will force the feature, empty or omitted values
```

Comment:
Think you need to say what the format of the items in the list is.

Reply:
I added an example. That's what I like to have when reading doc myself.